cmd/embedded-cluster/install.go

```diff
@@ -161,6 +161,11 @@ func RunHostPreflights(c *cli.Context, provider *defaults.Provider, applier *add
 		K0sDataDir:              provider.EmbeddedClusterK0sSubDir(),
 		OpenEBSDataDir:          provider.EmbeddedClusterOpenEBSLocalSubDir(),
 		SystemArchitecture:      runtime.GOARCH,
+		PrivateCA:               provider.PrivateCA(),
+		HTTPProxy:               provider.HttpProxy(),
+		HTTPSProxy:              provider.HttpsProxy(),
+		ProvidedNoProxy:         provider.ProvidedNoProxy(),
+		NoProxy:                 provider.NoProxy(),
```

Comment:
are you able to use the `proxy *ecv1beta1.ProxySpec` arg instead rather than the provider?

Reply:
yeah, that seems to work